Engine/util/library_windows.h

```diff
@@ -103,7 +104,11 @@ class WindowsLibrary final : public BaseLibrary
     {
         if (!_library)
             return nullptr;
+#if AGS_PLATFORM_WINDOWS_MINGW
+        return reinterpret_cast<void *>(GetProcAddress(_library, fn_name.GetCStr()));
+#else
```

Comment:
I think this does not need a platform switch, as this may be a valid type cast regardless.

Reply:
I removed the check and used the reinterpret_cast as the way to go there.

Reply:
I wonder if it could just have auto in the method itself to not have any cast and work with all implementations somehow, but I will look into this later - I don't remember if this was already a thing in c++11.